app/post.go

```diff
@@ -1379,6 +1375,11 @@ func (a *App) countThreadMentions(user *model.User, post *model.Post, teamId str
 		return int64(count), nil
 	}
 
+	team, err := a.GetTeam(teamId)
+	if err != nil {
+		return 0, err
+	}
```

Comment:
Would group channels still fail the check?

Reply:
I don't understand it 100%, but apparently, `Thread().GetMembershipsForUser` returns nil for GM, and therefore exits the condition before reaching `countThreadMentions`.

Reply:
My bad, you are right. It will also fail for GMs. Will fix.